lib/cdo/git_utils.rb

```diff
@@ -93,11 +93,7 @@ def self.pr_base_branch_or_default_no_origin
   end
 
   def self.circle_pr_branch_base_no_origin
```

Comment:
It looks like this function is used in other places (such as `eyes_steps.rb`), so probably not safe to change this way. Seems safest to leave this alone and use the Drone env variable directly in `branch_to_base`.

Reply:
Sorry I didn't take time to write up my analysis on the impact to other uses of this method.  Here's what I found:

This method stopped working (it always returns `nil` now) when we switched from CircleCI to Drone because while Drone supports many CircleCI environment variables for cross-compatibility, it does not support the `CI_PULL_REQUEST` environment variable that CirceCI deprecated in Circle 2.0.

The Eyes steps [no-op](https://github.com/code-dot-org/code-dot-org/blob/55d675b29f52096f2565b1285406eb059bb2270a/dashboard/test/ui/features/step_definitions/eyes_steps.rb#L15) in Continuous Integration (Circle/Drone) builds, so the call to this method in the "I open my eyes" step always returns `nil` both before and after this proposed change.

I'm confident that the Eyes step does not need to be updated to account for this change.

Reply:
The Eyes logic that uses this method is a legacy attempt to implement branching logic for Eyes Baseline images, which Eyes now officially supports with the ability to receive Webhooks from GitHub.  When we revive and successfully implement feature branches for Eyes baseline images, the section of Eyes steps that invoke this method will be cleaned up.

Reply:
Can you do a little cleanup based on your findings, then? It sounds like we can just delete the call to it and the branch of the if statement which uses it from `eyes_steps`.

Reply:
Yes!  Thanks for keeping me honest.  This is a good opportunity to pay off some technical debt.